docs/src/main/paradox/kinesis.md

```diff
@@ -135,6 +125,46 @@ Scala
 Java
 : @@snip [snip](/kinesis/src/test/java/docs/javadsl/KinesisSnippets.java) { #flow-sink }
 
+# AWS KCL Scheduler Source & checkpointer
+
+The KCL Source can read from several shards and rebalance automatically when other Schedulers are started or stopped. It also handles record sequence checkpoints.
+
+For more information about KCL please visit the [official documentation](http://docs.aws.amazon.com/streams/latest/dev/developing-consumers-with-kcl.html).
```

Comment:
The version 2 link seems to be https://docs.aws.amazon.com/streams/latest/dev/developing-consumers-with-kcl-v2.html